starport/templates/module/create/stargate/x/{{moduleName}}/keeper/keeper_test.go.plush

```diff
@@ -0,0 +1,33 @@
+package keeper
+
+import (
+	"testing"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+	"github.com/cosmos/cosmos-sdk/store"
+	storetypes "github.com/cosmos/cosmos-sdk/store/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/require"
+	"github.com/tendermint/tendermint/libs/log"
+	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
+	tmdb "github.com/tendermint/tm-db"
+	"<%= modulePath %>/x/<%= moduleName %>/types"
+)
+
+func setupKeeper(t testing.TB) (*Keeper, sdk.Context) {
```

Comment:
```suggestion
func newKeeper(t testing.TB) (*Keeper, sdk.Context) {
```

Reply:
tbh i think that setup is more descriptive